modules/ROOT/pages/tools/io-tool.adoc

```diff
@@ -0,0 +1,110 @@
+
+* Data Storage
+* Logging
+* Settings
+
+If you only see these three, it means that you are *not* yet connected to the agrirouter.
+
+== Data Storage
+
+This view allows you to select the data you intend to transfer.
+
+image::io-tool/datastorage.png[]
+
+You can either upload files from your device or select from the list of previously uploaded files. The system will 
+automatically select the file type, which can be manually adjusted later. Files are grouped by type in expandable sections.
+
+//TODO-Frank please provide some data I can upload and test (successful, unsuccessful)
+
+The lower panel contains the logs of all actions, which can also be expanded.
+
+== Logging
+
+The Logging section of the Main Menu provides more detailed information about the actions performed. 
+
+image::io-tool/logging.png[]
+
+// upload a screenshot later when more actions are performed
+
+== Getting Ready for Data Exchange
+
+To be able to use the IO-Tool for data exchange, you need to connect the IO-Tool to an agrirouter account. To do this, go to the Settings section in the main menu and select either MQTT or REST in the Production or QA (Quality Assurance) environment for the agrirouter connection. QA refers to the quality assurance of the agrirouter, not your application.
+
+When deciding between QA or Production, consider that new applications are only created in Production. Normally you would want to use MQTT because it is faster. Here you can find more information about https://docs.agrirouter.com/agrirouter-interface-documentation/latest/communication.html#mqtt[MQTT] and https://docs.agrirouter.com/agrirouter-interface-documentation/latest/communication.html#rest[REST].
+
+After selecting the protocol you will be taken to the *agrirouter login* view and need to confirm the action in this screen:
+
+(_Please note that after disconnecting from the agrirouter account you will need to wait for ca. 5 minutes to be able to connect again.`_)
+
+image::io-tool/connect.png[]
+
+Please be patient as this may take a while to get connected.
+
+== Sending Data
+
+After being successfully connected to the agrirouter account you will have two new sections in your main menu (Endpoints & Simulation) and more advanced settings.
+
+image::io-tool/menu.png[]
+
+== Endpoints
+
+To be able to send data you need to have *endpoints*. The endpoints are the representation of the farmer's software components in the agrirouter account that your IO-Tool account is connected to. Such components are e.g. a terminal, which is a communication unit (CU), or a farm management system, which is a farming software.
+In order to exchange data, you must therefore have at least one second IO-Tool account connected to the same agrirouter account. To connect it, just repeat the onboarding steps.
+
+
+// Add a link
+
+Once you have connected your second IO-Tool account to the *same* agrirouter account, you will need to switch to the agrirouter to set routes.
```

Comment:
At this point we should link the End User documentation where Routings are described and the Developers Documentation where Routings are described. 

Important is the remark that the necassity( right word?) to set this routings is a special case for FarmingSoftware to farmingSoftware. We should point this out really really explicitly as this leads to confusion very often